doc/doxygen/user_manual/app_files/app_files.md

```diff
@@ -0,0 +1,537 @@
+
+//n3
+scalarvalueType n3 = variable_list.get_scalar_value(3);
+
+//u
+vectorgradType Dux = variable_list.get_change_in_vector_gradient(4);
+
+
+// --- Setting the expressions for the terms in the governing equations ---
+
+vectorgradType eqx_Du;
+
+// Interpolation functions
+
+scalarvalueType h1V = (10.0*n1*n1*n1-15.0*n1*n1*n1*n1+6.0*n1*n1*n1*n1*n1);
+scalarvalueType h2V = (10.0*n2*n2*n2-15.0*n2*n2*n2*n2+6.0*n2*n2*n2*n2*n2);
+scalarvalueType h3V = (10.0*n3*n3*n3-15.0*n3*n3*n3*n3+6.0*n3*n3*n3*n3*n3);
+
+// Take advantage of E being simply 0.5*(ux + transpose(ux)) and use the dealii "symmetrize" function
+dealii::Tensor<2, dim, dealii::VectorizedArray<double> > E;
+E = symmetrize(Dux);
+
+// Compute stress tensor (which is equal to the residual, Rux)
+if (n_dependent_stiffness == true){
+	dealii::Tensor<2, CIJ_tensor_size, dealii::VectorizedArray<double> > CIJ_combined;
+	CIJ_combined = CIJ_Mg*(constV(1.0)-h1V-h2V-h3V) + CIJ_Beta*(h1V+h2V+h3V);
+
+	computeStress<dim>(CIJ_combined, E, eqx_Du);
+}
+else{
+	computeStress<dim>(CIJ_Mg, E, eqx_Du);
+}
+
+// --- Submitting the terms for the governing equations ---
+
+variable_list.set_vector_gradient_term_LHS(4,eqx_Du);
+
+}
+```
+
+Like the functions for the RHS, equationLHS takes variables_list and q_point_loc as inputs. However, because only one TIME_INDEPENDENT equation is solved at a time, the output are the terms for a single governing equation. As in explicitEquationRHS and nonExplicitEquationRHS, the model variable values and derivatives are given convenient names at the start of the file. The equationLHS function has a new option for getting variable values and derivatives, here one can access the change in the value of one of the variables or its derivatives. The function name is ```get_change_in_vector_gradient``` above. The other options are ```get_change_in_scalar_value```, ```get_change_in_scalar_gradient```, ```get_change_in_scalar_hessian```, ```get_change_in_vector_value```, and ```get_change_in_vector_hessian```.  Similar to the RHS functions, the value of eqx_Du is set and then used to set the appropriate residual for variables_list (in this case the vector residual term for the fourth variable).
+
+If multiple TIME_INDEPENDENT equations need to be solved, you will need to use conditional statements to calculate the proper residual depending on the elliptic equation being solved. The index of the field being solved can be accessed using the ```this-$>$currentFieldIndex``` statement.
+
+## ICs_and_BCs.cc
+The file ''ICs_and_BCs.cc'' contains the initial conditions for the primary variables as well as the expressions for non-uniform Dirichlet boundary conditions, if present. The file contains two functions: setInitialCondition and setNonUniformDirichletBCs.
+
+### setInitialCondition
+Here's a look at the setInitialCondition function for the coupled Allen-Cahn/Cahn-Hilliard example application:
+```
+template <int dim, int degree>
```

Comment:
do you want to leave in the [[maybe unused]] changes. Or any other template changes i may have missed.